models/public/densenet-161/densenet-161.md

```diff
@@ -6,11 +6,11 @@ The `densenet-161` model is one of the [DenseNet](https://arxiv.org/abs/1608.069
 group of models designed to perform image classification. The main difference with
 the `densenet-121` model is the size and accuracy of the model. The `densenet-161`
 is much larger at 100MB in size vs the `densenet-121` model's roughly 31MB size.
-Originally trained on Torch, the authors converted them into Caffe* format. All
-the DenseNet models have been pretrained on the ImageNet image database. For details
+Originally trained on Torch, the authors converted them into Caffe\* format. All
+the DenseNet models have been pre-trained on the ImageNet image database. For details
 about this family of models, check out the [repository](https://github.com/shicai/DenseNet-Caffe).
 
-The model input is a blob that consists of a single image of 1x3x224x224 in BGR
+The model input is a blob that consists of a single image of `1, 3, 224, 224` in `BGR`
```

Comment:
I don't think, that it is necessary to use `highlighting` in description section since it is copied to model.yml's description and \`this\` transforms to "this". But also won't insist on changes reverting.